Maple2.File.Ingest/Mapper/TableMapper.cs

```diff
@@ -585,6 +592,61 @@ public class TableMapper : TypeMapper<TableMetadata> {
         return new MasteryRewardTable(results);
     }
 
+    private FishingSpotTable ParseFishingSpot() {
+        var results = new Dictionary<int, FishingSpotTable.Entry>();
+        foreach ((int mapId, FishingSpot spot) in parser.ParseFishingSpot()) {
+            var entry = new FishingSpotTable.Entry(mapId, spot.minMastery, spot.maxMastery, spot.liquidType);
+            results.Add(mapId, entry);
+        }
+        return new FishingSpotTable(results);
+    }
+
+    private FishTable ParseFish() {
+        var results = new Dictionary<int, FishTable.Entry>();
+
+        // Parse habitat and combine
+        var habitats = new Dictionary<int, IReadOnlyList<int>>();
+        foreach ((int id, FishHabitat habitat) in parser.ParseFishHabitat()) {
+            var habitatMaps = new List<int>();
+            foreach (string map in habitat.habitat) {
+                string mapString = map.Replace(",", "");
```

Comment:
This seems weird, are there extra commas or something?

Reply:
There must be something with the parser on this one. It seems like it may be splitting the strings by space instead of a comma?